hazelcast/datadog_checks/hazelcast/check.py

```diff
@@ -0,0 +1,50 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import AgentCheck
+
+from . import utils
+
+
+class HazelcastCheck(AgentCheck):
```

Comment:
should this file by named `hazelcast.py`?

Reply:
No need imo